samples/Azure.Management.Storage/Generated/Models/SmbSetting.cs

```diff
@@ -31,7 +31,19 @@ internal SmbSetting(Multichannel multichannel, string versions, string authentic
         }
 
         /// <summary> Multichannel setting. Applies to Premium FileStorage only. </summary>
-        public Multichannel Multichannel { get; set; }
+        internal Multichannel Multichannel { get; set; }
+        /// <summary> Indicates whether multichannel is enabled. </summary>
+        public bool? MultichannelEnabled
+        {
+            get => Multichannel is null ? default : Multichannel.Enabled;
+            set
+            {
+                if (Multichannel is null)
+                    Multichannel = new Multichannel();
+                Multichannel.Enabled = value;
```

Comment:
Do we treat adding required property to `Multichannel` as a breaking change?

Reply:
If you add a required parameter to any method including ctor thats a breaking change and shouldn't be allowed